packages/slice-machine/lib/mock/misc/handlers.js

```diff
@@ -22,7 +24,8 @@ const handleFieldMock = (widget, maybeFieldMock = {}, config) => {
 
 export const handleFields = (Widgets) => (fields = [], mocks = {}) => {
   return fields.reduce((acc, [key, value]) => {
-    const widget = Widgets[value.type]
+    const typeName = value.type
+    const widget = typeName === 'Link' && value.config?.select === 'media' ? LinkToMediaMockWidget : Widgets[typeName]
```

Comment:
should linkToMedia be the only specific case or should we also have content relatonship as a special case?

In that case, I think we should use the function that was design to get the widget and handle specific case

Reply:
That's right